src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/HealthHeartbeatDefaultPayload.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    internal class HealthHeartbeatDefaultPayload : IHealthHeartbeatPayloadExtension
+    {
+        public const string FieldRuntimeFrameworkVer = "runtimeFramework";
+        public const string FieldTargetFramework = "targetFramework";
+        public const string FieldAppInsightsSdkVer = "appinsightsSdkVer";
```

Comment:
SDK version will be reported as sdkVersion field. No need to reserve an additional property for it

Reply:
Got it, SDK version is removed.